src/python/grpcio/grpc/__init__.py

```diff
@@ -721,6 +768,26 @@ def peer(self):
     raise NotImplementedError()
 
   @abc.abstractmethod
+  def parent(self, child=None, census=None):
+    """Creates a value to associate invoked RPCs with the current RPC.
+
+    This method's parameters have no guaranteed defaults; if either True or
+    False is desired for the value of an argument the argument should be
+    explicitly passed.
+
+    Args:
+      child: An optional boolean indicating whether or not the invoked RPC
```

Comment:
At the Core API level, census information only gets propagated by a call to `grpc_census_call_set_context`, and from the way that API looks I gather that census propagation is completely independent of parent-child relationship. That's why they're two independent parameters in this method.

I don't understand what you mean by "root"?